guides/shipments/shipping.md

```diff
@@ -1,129 +1,194 @@
+  cancelled. When this happens, all items in the shipment will be restocked. If
+  an order is "resumed", then the shipment will also be resumed.
 
-### Shipping Methods
+## Core concepts
 
-Shipping methods identify the actual services or carriers used to ship the product. For example:
+To leverage Solidus's shipping system, become familiar with its key concepts:
 
-* UPS Ground
-* UPS One Day
-* FedEx 2Day
-* FedEx Overnight
-* DHL International
+- Stock locations
+- Shipping methods
+- Zones
+- Shipping categories
+- Shipping calculators (through shipping rates)
 
-Each shipping method is only applicable to a specific geographic **Zone**. For example, you wouldn't be able to get a package delivered internationally using a domestic-only shipping method. You can't ship from Dallas, USA to Rio de Janeiro, Brazil using UPS Ground (a US-only carrier).
+### Stock locations
 
-If you are using shipping categories, these can be used to qualify or disqualify a given shipping method.
+Stock locations represent physical storage locations from which stock is
+shipped.
 
-**Note**: *Shipping methods can have multiple shipping categories assigned to them. This allows the shipping methods available to an order to be determined by the shipping categories of the items in a shipment.*
+### Shipping methods
+
+Shipping methods identify the actual delivery services used to ship the
+product. For example:
+
+- UPS Ground
+- UPS One Day
+- FedEx 2Day
+- FedEx Overnight
+- DHL International
+
+Each shipping method is only applicable to a specific geographic
+zone. For example, you wouldn't be able to get a package delivered
+internationally using a domestic-only shipping method. You can't ship from
+Dallas, USA, to Rio de Janeiro, Brazil, using UPS Ground, which is a United
+States-only carrier.
+
+### Shipping categories
+
+You can further restrict when shipping methods are available to customers by
+using shipping categories.
+
+For example, if your store can only ship oversized products via a specific
+carrier, called "USPS Oversized Parcels", then you could create a shipping
+category called "Oversized" for that shipping method, which can then be assigned
+to oversized products.
+
+Shipping categories are created in the admin interface (**Settings -> Shipping
+-> Shipping Categories**) and then assigned to products (**Products -> Edit**).
 
 ### Zones
 
-Zones serve as a mechanism for grouping geographic areas together into a single entity. You can read all about how to configure and use Zones in the [Zones Guide](addresses#zones).
+Zones serve as a mechanism for grouping geographic areas together into a single
+entity.
 
-The Shipping Address entered during checkout will define the zone the customer is in and limit the Shipping Methods available to him.
+The shipping address entered during checkout defines the zone the customer is
+in. This limit the available shipping methods and defines regional taxation
```

Comment:
I would take away taxation from here, as it's not really relevant to this guide.